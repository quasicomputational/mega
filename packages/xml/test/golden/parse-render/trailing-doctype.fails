<foo>hello</foo>
<!DOCTYPE foo>
