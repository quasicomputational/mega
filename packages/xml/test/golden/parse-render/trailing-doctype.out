Error: Stray doctype at (2, 1).
